official/Mask_RCNN/mehdi_resnet_model.py

```diff
@@ -0,0 +1,574 @@
+#resnet 50 and 101 v1
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+## models/officials/resnet/resnet_model ########################################################3
+_BATCH_NORM_DECAY = 0.997
+_BATCH_NORM_EPSILON = 1e-5
+DEFAULT_VERSION = 2
+DEFAULT_DTYPE = tf.float32
+CASTABLE_TYPES = (tf.float16,)
+ALLOWED_TYPES = (DEFAULT_DTYPE,) + CASTABLE_TYPES
+
+FEATURE_PYRAMID_DIMENSION = 256
+
```

Comment:
Code style note: keep two blank lines in between functions (here and below). In general for style questions refer to https://google.github.io/styleguide/pyguide.html